app/lib/zooniverse-logging.js

```diff
@@ -1,4 +1,5 @@
 const enabledTokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy', 'mschwamb/comet-hunters', 'zooniverse/snapshot-wisconsin'];
```

Comment:
does this need to stick around now?

Reply:
i have a bad habit of letting things stick around for historical purposes in spite of the existence of source control. thank you for the reminder :)

Reply:
(this has been fixed)